RedfishPkg/RedfishHttpDxe/RedfishHttpDxe.c

```diff
@@ -92,6 +92,67 @@ RedfishRetryRequired (
   return FALSE;
 }
 
+/**
+
+  This function follows below sections in Redfish specification to
+  check HTTP status code and see if this is success response or not.
+
+  7.5.2 Modification success responses
+  7.11 POST (action)
+
+  @param[in]  Method          HTTP method of this status code.
+  @param[in]  StatusCode      HTTP status code.
+
+  @retval     BOOLEAN         Return true when this is success response.
+                              Return false when this is not success response.
+
+**/
+BOOLEAN
+RedfishSuccessResponse (
+  IN EFI_HTTP_METHOD       Method,
+  IN EFI_HTTP_STATUS_CODE  *StatusCode
+  )
+{
+  BOOLEAN  SuccessResponse;
+
+  if (StatusCode == NULL) {
+    return TRUE;
+  }

```

Comment:
I think it is fine because this function focus on checking modification response. Returning true here to prevent false alarm as we have no status code to check.